### DISCUSSION TITLE
Etherscan Plugin should create all valid contracts in the contracts array even if there are a few unverified contracts

### DISCUSSION THREAD

**Discussion by Amirjab21** [wevm/wagmi#1807] [Ideas]
I've created a big array of contracts that I'm passing into the etherscan plugin for it to generate hooks.

One or two of these contracts are not verified and therefore they make the whole process fail.

It would be nice if it still created hooks for the verified contracts and just gave a warning about the ones that aren't verified.

**Comment by tmm** [comment]
Good idea! We could add a `filterUnverified` option to the Etherscan Plugin that does this. Anyone welcome to pick this up, otherwise I can add to my backlog.

Edit: Ideally this would be added to the Block Explorer Plugin and Etherscan Plugin could simply pass through the option.
